src/amo/components/Addon/index.js

```diff
@@ -73,9 +80,13 @@ export class AddonBase extends React.Component {
   }
 
   componentWillMount() {
```

Comment:
Thanks; sorry it was an awkward place to leave a comment as I know it won't be cleared. 😅